synapse/tests/test_lib_link.py

```diff
@@ -26,14 +26,23 @@ async def _spawnHost(n, pipe):
     link0, sock0 = await s_link.linksock()
     info = await link0.getSpawnInfo()
     pipe.send(info)
-    data = sock0.recv(n)
+    buf = b''
+    j = n
+    while True:
+        data = sock0.recv(j)
+        buf = buf + data
+        if len(buf) == n:
+            break
+        j = n - len(data)
 
     sock0.close()
     await link0.fini()
 
-    if data == b'V' * n:
+    if buf == b'V' * n:
         return
 
+    print(f'Data is not equal to V*{n}, got {len(buf)} {buf[:30]}...')
```

Comment:
stray print...

Reply:
Actually this is in failure case. If it fails again in the spawned process it'll be more evident.